### DISCUSSION TITLE
Batch write json files

### DISCUSSION THREAD

**Discussion by Uniky** [Mikubill/sd-webui-controlnet#2029] [Q&A]
**Is there a way to batch-write open pose preprocessor results in JSON?**

![image](https://github.com/Mikubill/sd-webui-controlnet/assets/24896456/e8469889-2887-42da-804d-82eab42b90f3)

**Comment by huchenlei** [comment]
https://github.com/Mikubill/sd-webui-controlnet/pull/2033 adds JSON pose data to API response. You can now batch get the JSON using the API.
